_til/linux/install-ubuntu.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Install Ubuntu 
+category: linux
+---
+
+Do you want to move from Windows into a UNIX like dev environment? Ubuntu is an easy way to get your feet wet and try out Linux.
+
+## Basic Installation
+
+Ubuntu ships with a very straight forward installer and makes set up a snap. Here are the only steps:
+* Choose a language and keyboard layout (Usually English-US for both)
+* Connect to a network (Optional, convenient to install updates during installation)
+* Choose to install updates
+* Choose to install 3rd party non-free items (Flash, MP3, etc. I usually select yes as it makes the user experience much easier)
+* Create a user and hostname
+* Partition the disk (See below)
```

Comment:
Manual disk partitioning is typically not required. There are options to install over any existing OS (which will also apply to no OS) and to install along side the existing OS. This should be recommended unless more control is needed